scipy/integrate/_quad_vec.py

```diff
@@ -221,6 +221,23 @@ def quad_vec(f, a, b, epsabs=1e-200, epsrel=1e-8, norm='2', cache_size=100e6,
     >>> plt.ylabel(r"$\int_{0}^{2} x^\alpha dx$")
     >>> plt.show()
 
+    When using the argument ``workers``, one should ensure
```

Comment:
Sorry missed this last time around!
```suggestion
    When using the argument `workers`, one should ensure
```